erpnext/selling/doctype/quotation/quotation.py

```diff
@@ -185,6 +185,12 @@ def update_item(obj, target, source_parent):
 
 	return doclist
 
+def set_expired_status():
+	from datetime import date
+	DATE_FORMAT = "%Y%m%d" # For converting python date to SQL comparable date
+	today = date.today().strftime(DATE_FORMAT)
+	frappe.db.sql("UPDATE tabQuotation SET status = 'Expired' WHERE valid_till < " + today)
```

Comment:
```suggestion
	frappe.db.sql("""UPDATE `tabQuotation` SET status = 'Expired' 
	WHERE status != 'Expired' AND 'valid_till < %s""" , (nowdate()))
```
This should do I guess. No need to import date and use DATE_FORMAT